layers.go

```diff
@@ -772,7 +772,18 @@ func (r *layerStore) Mounted(id string) (int, error) {
 }
 
 func (r *layerStore) Mount(id string, options drivers.MountOpts) (string, error) {
-	if !r.IsReadWrite() {
+
+	// check whether options include ro option
+	hasReadWriteOpt := func(opts []string) bool {
+		for _, item := range opts {
+			if item == "ro" {
+				return false
+			}
+		}
+		return true
```

Comment:
Can we negate the logic and call it `hasReadOnlyOpt`? There's an implicit assumption that "no ro" is "read write" which would be easier to understand the other war around.

Reply:
The problem with that is we end up wit an awful double negative line below

I don't like that name, because you end up with a double negative when you use the function below.
I don;t like
```
if !r.IsReadWrite() && !hasReadOnlyOpt(options.Options) {
```

Reply:
The returns need to be inverted. I opened https://github.com/containers/storage/pull/667